### ISSUE TITLE
Web Directory

### ISSUE THREAD

**Issue by DonaldHandley** [openaudible/openaudible#1077]
How is the Web directory used and is it necessary if I use Plex?

Thanks

**Comment by openaudible** [comment]
No. It can be deleted. We are in the process of revamping how it works as well..
